### PR TITLE
refactor(@vtmn/css): apply new styles for button component

### PR DESCRIPTION
**Pull request by GaspardMathon**
## Pull Request checklist

🚨 Please review the [guidelines for contributing](../blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Don't request your main!
- [x]  Make sure you are making a pull request against the **main branch** (left side).
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.

## Description

- Change colors for most of the variant
- Remove secondary-reversed
- Add a ghost-reversed and a tertiary variant

## Does this introduce a breaking change?

- No

❤️ Thank you!

### CODE REVIEW

File: packages/sources/css/src/components/button/src/index.css [hunk 1]
```diff
@@ -89,42 +89,59 @@
...
+  background-color: var(--vtmn-color_white);
+  box-shadow: inset 0 0 0 rem(2px) var(--vtmn-color_grey-light-2);
+  color: var(--vtmn-color_brand-digital-dark-1);
+  fill: var(--vtmn-color_brand-digital-dark-1);
 }
 
 .vtmn-btn_variant--secondary:not([disabled]):hover {
-  background-color: color-mod(var(--vtmn-color_grey-dark-2) alpha(5%));
+  background-color: #f2f9fc;
 }
```
Comment (Margot-ElMadi) [inline packages/sources/css/src/components/button/src/index.css:110]: use variables

Comment (GaspardMathon) [reply packages/sources/css/src/components/button/src/index.css:110]: Thanks for your feedback, I added the new color variables for the light blue

**Review by lauthieb — APPROVED** [review]

File: packages/sources/css/src/components/button/src/index.css [hunk 2]
```diff
@@ -89,42 +89,59 @@
...
+.vtmn-btn_variant--conversion {
+  box-shadow: inset 0 0 0 rem(2px) transparent;
+  background-color: var(--vtmn-color_yellow);
+  color: var(--vtmn-color_black);
+  fill: var(--vtmn-color_black);
 }
 
-.vtmn-btn_variant--secondary-reversed:not([disabled]):hover {
-  background-color: color-mod(var(--vtmn-color_white) alpha(20%));
+.vtmn-btn_variant--conversion:not([disabled]):hover {
```
Comment (lauthieb) [inline packages/sources/css/src/components/button/src/index.css:139]: Breaking change, we cannot remove "secondary reversed" CSS class without making a major version. So for the moment, you can move secondary-reversed classes to the bottom of this file under a "Deprecated" comment.

**Review by lauthieb — CHANGES_REQUESTED** [review]

File: packages/sources/css/src/components/button/src/index.css (hunk 2 above)
Comment (GaspardMathon) [reply packages/sources/css/src/components/button/src/index.css:139]: Done

File: packages/sources/css/src/components/button/src/index.css [hunk 3]
```diff
@@ -302,3 +316,21 @@
 .vtmn-btn_size--large.vtmn-btn--icon-right svg {
   margin-left: rem(12px);
 }
+
+/* Deprecated */
```
Comment (lauthieb) [inline packages/sources/css/src/components/button/src/index.css:320]:
```suggestion
/* DEPRECATED */
```

**Review by lauthieb — APPROVED** [review]

**Pull request merged**